tests/testthat/helper-TealAppDriver.R

```diff
@@ -27,3 +27,34 @@ simple_cdisc_data <- function(datasets = c("ADSL", "ADRS", "ADTTE")) {
   teal.data::join_keys(data) <- teal.data::default_cdisc_join_keys[datasets]
   data
 }
+
+# local app drivers for module testing ----------------------------------------
+# based on examples
+
+local_app_tm_a_pca <- function() {
```

Comment:
@averissimo @vedhav @kartikeyakirar  I like the idea of this `local_app`. We have the same approach for `teal.widgets` https://github.com/insightsengineering/teal.widgets/blob/main/tests/testthat/helpers-with-settings.R

I would only remove `local` from the name.
What do you think about updating other PRs like this one : ) ?

Reply:
I :heart: it, but I'm heavily biased towards it as I love `{withr}`

Reply:
renamed to `app_driver_tm_a_pca`